src/core/models.py

```diff
@@ -863,19 +863,25 @@ def has_missing_image_files(self):
                 # attempt to pull a URL from the specified attribute
                 url = os.path.basename(val.get(attribute, None))
 
-                try:
-                    try:
-                        self.images.get(original_filename=url)
-                    except File.MultipleObjectsReturned:
-                        self.images.filter(original_filename=url).first()
-                except File.DoesNotExist:
+                if show_all:
                     missing_elements.append(url)
+                else:
+                    try:
+                        try:
+                            self.images.get(original_filename=url)
+                        except File.MultipleObjectsReturned:
+                            self.images.filter(original_filename=url).first()
+                    except File.DoesNotExist:
```

Comment:
This should be at the same level as the previous `except File.MultipleObjectsReturned:` instead of wrapping it

Reply:
Dropping this as suggested above.